dts/bindings/input/input-evdev.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Linux evdev based input device
+
+  Allows using a Linux evdev device to read event and report them back as
+  Zephyr input events.
+
+  Example configuration:
+
+  evdev {
+          compatible = "input-evdev";
+  };
+
+  Then run the binary specifying the evdev device with the --evdev flag, for
+  example:
+
+  ./build/zephyr/zephyr.exe --evdev=/dev/input/event0
```

Comment:
I guess the user will need to sudo to be able to read an input device? (anyhow, better described in docs :) )

Reply:
Ahh... maybe? :-) I don't, guess I added my user to the input group on all my systems. Honestly I'd rather leave this up to the user to figure out rather than adding a Linux system howto in every driver